### PR TITLE
Add tags for 404 and 500 messages.

### PR DESCRIPTION
**Pull request by MichaelWalker**
To do (delete all that do not apply):

 - [x] Changelog entry added.

### CODE REVIEW

**Comment by MichaelWalker** [comment]
image

File: demo/templates/demo/index.html [hunk 1]
```diff
@@ -45,17 +45,18 @@ <h3 class="heading-small" id="components">Components:</h3>
...
 
+      <h3 class="heading-small" id="header-footer">Headers and footers</h3>
       <ul class="list list-bullet" aria-labelledby="header-footer">
-        <h3 class="heading-small" id="header-footer">Headers and footers</h3>
         <li><a class="link" href="{% url 'great-international-header-footer' %}">Great.gov.uk international header and footer</a></li>
         <li><a class="link" href="{% url 'great-domestic-header-footer' %}">Great.gov.uk domestic header and footer</a></li>
         <li><a class="link" href="{% url 'invest-header' %}">Great.gov.uk invest header (and international footer)</a></li>
       </ul>
 
+      <h3 class="heading-small" id="header-footer">Analytics</h3>
```
Comment (eadpearce) [inline demo/templates/demo/index.html]: duplicated ID

File: directory_components/templates/404.html [hunk 2]
```diff
@@ -1,20 +1,11 @@
...
-      <h1 class="heading-xlarge">This page cannot be found</h1>
-      <ul class="list list-bullet">
-          <li>If you entered a web address please check it’s correct.</li>
-          <li>If you followed a link here, <a class="link" href="{{ services_urls.contact_us }}">let us know</a>.</li>
-          <li><a class="link" href="{{ services_urls.exred }}">Go to the homepage</a></li>
-      </ul>
-    </div>
-  </div>
-</section>
+    {% content_404 feedback_url=services_urls.contact_us home_page_url=services_urls.exred %}
```
Comment (eadpearce) [inline directory_components/templates/404.html]: noice 👍

File: demo/templates/demo/index.html (hunk 1 above)
Comment (MichaelWalker) [reply demo/templates/demo/index.html]: oh yeah.

Yep as you say, has been a thing for ever, but good chance to fix it now! 😊

**Pull request merged**